buildpack-deps/README.md

```diff
@@ -53,7 +53,7 @@ You are invited to contribute new features, fixes, or updates, large or small;
 we are always thrilled to receive pull requests, and do our best to process them
 as fast as we can.
 
-Before you start to code, we recommend discussing your plans 
+Before you start to code, we recommend discussing your plans
```

Comment:
Correct, the whole "User Feedback" section is generated.  We need to adjust and drop the mailing-list insertion logic, since this is the result when there is no list (and just customize the 2-3 that use it).